wherehows-web/app/templates/navbar.hbs

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+<nav class="navbar navbar-inverse navbar-fixed-top {{if banners.isShowingBanners "navbar-top-offset"}}" role="navigation">
```

Comment:
Please see my overlay comment above? I think having an overlay rather than offsetting might be a better approach?